examples/filetable.py

```diff
@@ -15,80 +16,96 @@ def filetable_to_firefly(ffclient, topdir, pattern, recursive=True):
     ffclient: firefly_client.FireflyClient
         Instance of FireflyClient connected to a Firefly server
 
-    topdir: 'str'
+    topdir: "str"
         pathname for directory to search
 
-    pattern: 'str'
-        filename pattern to search for, e.g. '*.fits'
+    pattern: "str"
+        filename pattern to search for, e.g. "*.fits"
 
     recursive: `bool`
         Search all subdirectories recursively (default=True)
 
     Returns:
     --------
-    tbl_val: 'str'
+    tbl_val: "str"
         Descriptor of table that was uploaded to the Firefly server
 
     metadict: `dict`
         Dictionary of metadata items
 
     """
-    filelist = glob(topdir+'/**/' + pattern, recursive=recursive)
-    metadict = {'datasource': 'path'}
-    with tempfile.NamedTemporaryFile(mode='w+t',
-                                     delete=False,
-                                     suffix='.csv') as fd:
+    filelist = sorted(glob(topdir + "/**/" + pattern, recursive=recursive))
+    metadict = {"datasource": "path"}
+    with tempfile.NamedTemporaryFile(mode="w+t", delete=False, suffix=".csv") as fd:
         csv_writer = csv.writer(fd)
-        csv_writer.writerow(['number','name','path'])
+        csv_writer.writerow(["number", "name", "path"])
         for i, path in enumerate(filelist):
-            csv_writer.writerow([i, os.path.basename(path),
-                                 'file://'+path])
+            # Docker Firefly allows uploads from /external
+            csv_writer.writerow([i,
+                                 os.path.basename(path),
+                                 "file:///external" + path])
```

Comment:
Because I tested it locally, I had to change it to this:
```suggestion
                                 "file://" + path])
```

Like you said, it will be better if you can add `/external` conditionally. Or simply just let user supply it in their topdir path, making it runnable anywhere.

Reply:
I made the "/external" into an optional argument. I'm not sure if my extended help description makes sense! I refer to Firefly-in-Docker...

Reply:
Perfect! thanks